### PR TITLE
Fix QSPI NOR flash for rak5010_nrf52840

### PR DESCRIPTION
**Pull request by parthitce**
- Fix QSPI NOR flash settings in rak5010_nrf52840
- Add overlay and conf for littlefs samples

### CODE REVIEW

File: samples/subsys/fs/littlefs/boards/rak5010_nrf52840.overlay [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+&is25wp064a {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		lfs1_part: partition@0 {
+			label = "storage";
+			reg = <0x00000000 DT_SIZE_M(64)>;
```
Comment (nordicjm) [inline samples/subsys/fs/littlefs/boards/rak5010_nrf52840.overlay:34]: This flash chip is most definitely not 64MiB in size

**Review by nordicjm — CHANGES_REQUESTED** [review]

File: samples/subsys/fs/littlefs/boards/rak5010_nrf52840.overlay (hunk 1 above)
Comment (parthitce) [reply samples/subsys/fs/littlefs/boards/rak5010_nrf52840.overlay]: Thanks for pointing. It's 64Mega bits. Adjusted it. Please revisit.

**Review by nordicjm — APPROVED** [review]

**Comment by parthitce** [comment]
@henrikbrixandersen can this be tagged for 3.6 as it's a bug fix? Thanks.

**Review by de-nordic — DISMISSED** [review]

**Comment by MaureenHelm** [comment]
Tagging for 3.7 because the second patch is more than a bug fix.

**Review by nordicjm — CHANGES_REQUESTED** [review]
Needs converting to hwmv2

**Review by nordicjm — APPROVED** [review]

**Review by de-nordic — APPROVED** [review]

**Pull request merged**